assets/sass/imports/_presentations.scss

```diff
@@ -0,0 +1,48 @@
+.presentation {
+    position: relative; 
+    display: flex; 
+    flex-direction: column;
+    justify-content: start;  
+    @media (max-width: 1340px) {
+        margin-left:0%;
+        margin-right:0%;
+    }
+}
+
+.presentation .presentation_section {
+    display: flex; 
+    flex-direction: row; 
+    justify-content: space-between; 
+    width: 100%;
+    flex-flow: wrap;
+}
+
+.presentation .presentation_section .info-box {
+    display: flex;
+    justify-content: center; 
+    align-items:center; 
+    width: 100%;
+    order: 1;
+}
+
+.presentation .presentation_section .preview-box {
+    width: 100%;
+    order: 2;
+    padding-bottom: 7%;
```

Comment:
```suggestion
```
See my previous comment about why padding margin shouldn't be in percent but in px. Also it's better to use margin when our intent is to create external spacing (out of box) - padding is meant for internal spacing. 

So to replace this, add `margin-bottom: 45px` in `.presentation` - I've tried it in web inspector, it will do the job.